OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -64,10 +61,10 @@ enum ChatPanelType { Lobby, Global }
 		readonly TabCompletionLogic tabCompletion = new TabCompletionLogic();
 
 		readonly LabelWidget chatLabel;
-		bool teamChat;
 
+		MapPreview Map { get; set; }
```

Comment:
LobbyLogic.Map was used in LobbyUtils so public get. I pass now Map instead of LobbyLogic so public was removed. I'll change it to field.